doc/rados/operations/balancer.rst

```diff
@@ -34,6 +34,10 @@ The balancer mode can be changed from ``upmap`` mode to ``crush-compat`` mode.
 ``crush-compat`` mode, the balancer automatically makes small changes to the
 data distribution in order to ensure that OSDs are utilized equally.
 
+Additional modes include ``upmap-read`` and ``read``. ``upmap-read`` mode
+combines the upmap balancer with the read balancer so writes and reads to allow
```

Comment:
read balancer so that both writes and reads are optimized.

Reply:
Fixed